crates/bevy_ecs/src/entity/mod.rs

```diff
@@ -162,6 +162,7 @@ pub struct Entities {
     /// Once `flush()` is done, `free_cursor` will equal `pending.len()`.
     pending: Vec<u32>,
     free_cursor: AtomicI64,
+    /// Stores the number of free entities for [`len`](Entities::len)
```

Comment:
wouldn't that `Entities::len` link just be a link to itself?

Reply:
I don't think so? There's a function also called len, which this should point to

I can check though at some point. It's only an internal documentation anyway, since it's on a private field.

Reply:
Oh I didn't see the method with the same name in this diff 👍 

Anyway I can't get rust-analyser to correctly link to something from `Entities`, not sure why...

Reply:
`rust-analyser`'s intra doc links syntax is a subset of valid links, from what I've found.

The way to try this is with `cargo doc -- --document-private-items`  (or whatever the precise correct syntax is)

Reply:
I have confirmed that it links correctly

There are several other bugs which running `cargo doc --open --document-private-items` makes appear, but that's not the job of this PR.